scanpopup.cc

```diff
@@ -580,7 +580,7 @@ void ScanPopup::engagePopup( bool forcePopup, bool giveFocus )
     //QApplication::processEvents(); // Make window appear immediately no matter what
   }
   else
-  if ( ui.pinButton->isChecked() )
+  if ( ui.pinButton->isChecked() && cfg.preferences.raiseMainWindow)
   {
     // Pinned-down window isn't always on top, so we need to raise it
     show();
```

Comment:
`show()`   should this method be called?

Reply:
> show() should this method be called at all conditions?

I think so. I will change that.